### ISSUE TITLE
[Feature Request] Arguments

### ISSUE THREAD

**Issue by Qwerty-Space** [aparrish/pronouncingpy#44]
Hello,

I'd like to request arguments for `phones_for_word` such as `stress_marks=False`.  That would be brilliant.

And also the ability to convert multiple words in a single string.  Perhaps a `for_phone_in_words` function

**Comment by aparrish** [comment]
These are great suggestions, thank you! In the first case, it's easy enough to implement on your own, something like:

```python
>>> import pronouncing, re
>>> [re.sub(r"\d", "", item) for item in pronouncing.phones_for_word("perfect")]
['P ER F EH K T', 'P ER F IH K T']
```

In the second case, I'm hesitant to introduce word tokenization logic into the library, since it's such a can of worms, and I feel like most of the bug reports I would get would be about the word tokenization algorithm being wrong or insufficient (instead of what the library is supposed to be about, e.g., getting pronunciation information about words). I'd encourage users instead to make a decision about their own method of tokenizing words for their particular use case and work from there. E.g., with `nltk`:

```python
>>> import nltk
>>> ' '.join([pronouncing.phones_for_word(item)[0] for item in nltk.word_tokenize("this is a test!") if item in pronouncing.lookup])
'DH IH1 S IH1 Z AH0 T EH1 S T'
```

Though on second thought that is kind of a mouthful. I'll take it into consideration and would also be happy to consider a pull request with this feature...

**Comment by Acryptarch** [comment]
Is this module still being developed? I feel like I could implement the feature described

**Comment by aparrish** [comment]
> Is this module still being developed? I feel like I could implement the feature described

Yes! It is still being developed. As noted above, I'm willing to entertain PRs for these features, but I'm more inclined to leave these as downstream implementation tasks for users of the package, rather than incorporating them into the package itself.

**Comment by Acryptarch** [comment]
That's a fair point. What other features do you think would be useful? Maybe an implementation of the logios tool in python would be a good addition to this library.
